src/platform/silabs/multi-ota/OTAFactoryDataProcessor.h

```diff
@@ -76,6 +74,8 @@ class OTAFactoryDataProcessor : public OTATlvProcessor
     OTAFactoryPayload mPayload;
     OTADataAccumulator mAccumulator;
     uint8_t * mFactoryData = nullptr;
-};
 
+protected:
+    uint32_t GetAccumulatorSize() const override { return mLength; }
```

Comment:
mLength seems to be a member of the parent class. If it is, the function should be protected in the parent class

Reply:
Yes mLength is member of the parent class but parent class has implementation  `GetAccumulatorSize` which returns `sizeof(Descriptor)`.
 virtual uint32_t GetAccumulatorSize() const { return sizeof(Descriptor); }
which is common for all derived classes except `OTAFactoryDataProcessor `.
For this class OTAFactoryDataProcessor , have overridden this function to return mLength.